kiwi/partitioner/dasd.py

```diff
@@ -42,26 +42,31 @@ def post_init(self):
             't.csm': None
         }
 
-    def create(self, name, mbsize, type_name, flags=None):
+    def create(self, name, mbsize, type_name, flags=None, start_sector=None):
         """
         Create DASD partition
 
         :param string name: partition name
         :param int mbsize: partition size
         :param string type_name: unused
         :param list flags: unused
+        :param int start_sector: first sector in case it is the first partition
         """
         self.partition_id += 1
         fdasd_input = NamedTemporaryFile()
+        if self.partition_id > 1:
+            start_sector = None
         with open(fdasd_input.name, 'w') as partition:
             log.debug(
                 '%s: fdasd: n p cur_position +%sM w q',
                 name, format(mbsize)
             )
-            if mbsize == 'all_free':
-                partition.write('n\np\n\n\nw\nq\n')
-            else:
-                partition.write('n\np\n\n+%dM\nw\nq\n' % mbsize)
+            partition.write(
+                'n\np\n{0}\n{1}\nw\nq\n'.format(
+                    '' if not start_sector else start_sector,
+                    '' if mbsize == 'all_free' else '+{0}M'.format(mbsize)
+                )
+            )
```

Comment:
A DASD device does not allow a custom start sector. s390 will not use a dasd with any other than the default start sectors. If you haven't tested s390 booting from dasd with custom start sectors any different than what fdasd does and it worked, please revert that change and leave s390 untouched for the moment

Reply:
Ok, yes I haven't tested DASD, just verified that the fdasd call was the same without any custom start sector. You are right, better leave it untouched.